homeassistant/components/fritz/__init__.py

```diff
@@ -54,13 +47,6 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     if DATA_FRITZ not in hass.data:
         hass.data[DATA_FRITZ] = FritzData()
 
-    @callback
-    def _async_unload(event: Event) -> None:
-        fritz_tools.async_unload()
-
-    entry.async_on_unload(
-        hass.bus.async_listen_once(EVENT_HOMEASSISTANT_STOP, _async_unload)
-    )
     entry.async_on_unload(entry.add_update_listener(update_listener))
 
```

Comment:
```suggestion

    await fritz_tools.async_config_entry_first_refresh()

```
you need to do a first data fetch, so that `device_tracker` platform will setup properly